library/core-components/components/text-and-image/text-and-image.tsx

```diff
@@ -41,7 +41,7 @@ export const TextAndImage = forwardRef<React.ElementType, TextAndImageProps>(
             // @ts-expect-error - this component needs refactor with new tokens
             font="--typography-heading-lg"
             // @ts-expect-error - this component needs refactor with new tokens
-            color="--color-text-on-base-primary"
+            fill="--color-text-on-base-primary"
```

Comment:
Just fixing a type error - will be refactoring this component in the near future